pages/index.tsx

```diff
@@ -43,56 +39,6 @@ const Home: NextPage = () => {
-              <h2 className="md:mt-0 text-primary-blue">{t(`homepage.box_child_section_title`)}</h2>
-              <div className="sm:grid grid-cols-2 md:grid-cols-1 gap-8">
-                <div className="homepage-box w-full group hover:text-primary-blue    pr-4">
-                  <ChildStoriesIcon className="w-6 mb-2 group-hover:fill-primary-red" />
-                  <h2 className="text-lg mt-0 pb-1 inline-block border-b-1 border-b-solid border-b-primary-black">
-                    <Link href={'/kids/stories'}>
-                      <a>{t('homepage.box_child_stories_title')}</a>
-                    </Link>
-                  </h2>
-                  <p>{t(`homepage.box_child_stories_description.${getCountryVariant()}`)}</p>
-                </div>
-                <div className="homepage-box w-full group hover:text-primary-blue    pr-4">
-                  <MemoryGameIcon className="w-6 mb-2 group-hover:fill-primary-red" />
-                  <h2 className="text-lg mt-0 pb-1 inline-block border-b-1 border-b-solid border-b-primary-black">
-                    <Link href={'/kids/memory-game'}>
-                      <a>{t('homepage.box_child_memory_title')}</a>
-                    </Link>
-                  </h2>
-                  <p>{t(`homepage.box_child_memory_description`)}</p>
-                </div>
-                <div className="homepage-box w-full group hover:text-primary-blue    pr-4">
-                  <ChildIcon className="w-6 mb-2 group-hover:fill-primary-red" />
-                  <h2 className="text-lg mt-0 pb-1 inline-block border-b-1 border-b-solid border-b-primary-black">
-                    <Link href={'/kids#rytir'}>
-                      <a>{t('homepage.box_child_words_title')}</a>
-                    </Link>
-                  </h2>
-                  <p>{t(`homepage.box_child_words_description.${getCountryVariant()}`)}</p>
-                </div>
-                <div className="homepage-box w-full group hover:text-primary-blue    pr-4">
-                  <ChildDictionaryIcon className="w-6 mb-2 group-hover:fill-primary-red" />
-                  <h2 className="text-lg mt-0 pb-1 inline-block border-b-1 border-b-solid border-b-primary-black">
-                    <Link href={t('homepage.box_child_dictionary_link')}>
-                      <a>{t('homepage.box_child_dictionary_title')}</a>
-                    </Link>
-                  </h2>
-                  <p>{t(`homepage.box_child_dictionary_description`)}</p>
-                </div>
-              </div>
-            </div>
-            <div className="child-section__image md:col-start-2 md:col-end-4 md:row-start-1 md:row-end-2">
-              <Image src={ChildSectionImg} alt={t(`homepage.box_section_image_author`)} />
-              {t(`homepage.box_child_section_image_author`)}
-            </div>
-          </div>
-        </div>
         {/* Standard section */}
         <div className="max-w-7xl bg-white p-4 sm:p-8 md:p-12 shadow-xxl mb-8">
```

Comment:
Koukám, teď je ten prázdný prostor pod nadpisem "Naučte se základy češtiny a ukrajinštiny pro běžné životní situace" moc velký.

Koukám na https://github.com/cesko-digital/movapp/pull/141/files#diff-aa98fd0757d0e1741503c50cfafb7726939d19819638bbe8e030a27adfec34a3L42 tehdy tam bylo nějaké `classname mt-[-6rem]` které nejspíš tu mezeru zmenšovalo.

Reply:
To jsem úplně přehlédl, díky. Vrátil jsem to zpět a vypadá to líp 👍